rust/src/lib.rs

```diff
@@ -180,34 +212,42 @@ Accept-Encoding: gzip,deflate\r
             let (header, value) = split2(&line, ":").ok_or(MALFORMED_RESPONSE)?;
             let header = header.to_ascii_lowercase();
             let value = value.trim();
-            if header == "content-encoding" {
-                encoding = value.parse().map_err(|_| UNSUPPORTED_ENCODING)?;
-            }
             headers.insert(header, value.to_string());
         }
 
-        // 11. Read body
```

Comment:
I feel like this `11. Read body` comment is intended to be alive somewhere for educational purpose but it is now deleted.